Backpack-SwiftUI/CardCarousel/Classes/BPKCardCarousel.swift

```diff
@@ -66,23 +68,58 @@ internal struct InternalCardCarousel<Content: View>: View {
         content: [Content],
         currentIndex: Binding<Int>
     ) {
+        self.isIPad = UIDevice.current.name.hasPrefix("iPad")
```

Comment:
Using the device name is brittle and will break in the future. 
We should look into size classes: https://developer.apple.com/documentation/swiftui/userinterfacesizeclass

Especially since the app can be resized on iPad (or stage manager)